CONTRIBUTING.md

```diff
@@ -0,0 +1,262 @@
+# Contributing to pymovements
+Thank you for taking your time to contribute to pymovements!
+
+We encourage you to report any bugs or contribute to new features, optimisations or documentation.
+
+Here we give you an overview of the workflow and best practices for contributing
+to pymovements.
+
+**Questions:** If you have any developer-related questions, please [open an issue](
+https://github.com/aeye-lab/pymovements/issues/new/choose) or write us at
+[daniel.krakowczyk@uni-potsdam.de](mailto:daniel.krakowczyk@uni-potsdam.de).
```

Comment:
I'd recommend only to provide issue tracker. it would get rid of duplicates and unnecessary work (for you)

Reply:
I thought maybe some people will hesitate if it is that public?

Reply:
yea but they shouldn't... it could lead to a lot more work for you -- maybe we can make each private inquiry into an "anonoymous" issue?

Reply:
Yes you're right they really shouldn't be afraid of that..

Right now I don't think that it will be much more work, but we will see in the future.
And yes, in the end all private inquiries will have to go into the issue tracker either way.
I hope they will be able to do that themselves after being told so.
I just don't want to create bigger hurdles than necessary to get any feedback from potential users.

I am also in the process of creating a mailing list for us, because having a way of contacting us besides creating issues or questions here on github is still necessary I think. I will replace this with that list email address then.